awx/main/access.py

```diff
@@ -588,17 +588,41 @@ def can_delete(self, obj):
 
 
 class InstanceGroupAccess(BaseAccess):
+    """
+    I can see Instance Groups when I am:
+       - a superuser(system administrator)
+       - an organization admin
+       - at least read_role on the instance group
+    I can edit Instance Groups when I am:
+       - a superuser
+       - an organization admin
```

Comment:
current logic does not allow org admins to edit and Instance Group.

Reply:
Yes that is correct, I'll fix that up